packages/ui/context/MultiMidasContext.tsx

```diff
@@ -0,0 +1,44 @@
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { MidasSdk } from '@midas-capital/sdk';
+import { SupportedChains } from '@midas-capital/types';
+import { createContext, ReactNode, useContext, useMemo } from 'react';
+
+import { chainIdToConfig } from '../types/ChainMetaData';
+
+export interface MultiMidasContextData {
+  sdks: Partial<Record<SupportedChains, MidasSdk>>;
+}
+
+export const MultiMidasContext = createContext<MultiMidasContextData | undefined>(undefined);
+
+interface MultiMidasProviderProps {
+  children: ReactNode;
+}
+
+const ENABLED_CHAINS: SupportedChains[] = [56, 137];
```

Comment:
This is pretty cool. Should we add Moonbeam already, and also, use the @types module to get the ChainIds?